typing/typecore.ml

```diff
@@ -4665,7 +4665,7 @@ and type_function
                     [type_argument] on the cases, and discard the cases'
                     inferred type in favor of the constrained type. (Function
                     cases aren't inferred, so [type_argument] would just call
-                    [type_expect] straightaway, so we do the same here.)
+                    [type_expect] straight away, so we do the same here.)
```

Comment:
"straightaway" is perfectly correct, although rather British.  Americans would write "right away", I guess.

Reply:
Back to straightaway.